superset-frontend/src/components/ReportModal/HeaderReportActionsDropdown/index.tsx

```diff
@@ -122,16 +110,16 @@ export default function HeaderReportActionsDropDown({
         }),
       );
     }
-  }, [dashboardId]);
+  }, []);
 
   const menu = () => (
     <Menu selectable={false} css={{ width: '200px' }}>
       <Menu.Item>
         {t('Email reports active')}
         <Switch
           data-test="toggle-active"
-          checked={report?.active}
-          onClick={(checked: boolean) => toggleActiveKey(report, checked)}
+          checked={findReport?.active}
```

Comment:
is there another var referring to report? `findReport` sounds more like an action than an object. Can we use `report` instead?